packages/api-request-builder/src/build-query-string.js

```diff
@@ -57,8 +57,7 @@ export default function buildQueryString (
       queryString.push('fuzzy=true')
     if (fuzzyLevel)
       queryString.push(`fuzzyLevel=${fuzzyLevel}`)
-    if (markMatchingVariants)
-      queryString.push('markMatchingVariants=true')
+    queryString.push(`markMatchingVariants=${markMatchingVariants.toString()}`)
```

Comment:
Right here https://github.com/commercetools/nodejs/blob/master/packages/api-request-builder/src/default-params.js#L51

It was already there but query builder was sending just a `true` value.